### DISCUSSION TITLE
How to restore HLS file name structure change in Ant Media Server newer versions?

### DISCUSSION THREAD

**Discussion by USAMAWIZARD** [ant-media/Ant-Media-Server#4984] [FAQs] [answered]
I have been using Ant Media Server and one of our major use case is HLS playback.
Until the latest update to v2.4.1 the filename structure for HLS was like streamId_480p.m3u8
But after updating to v2.4.1+ the file name structure has changed to streamId_480p500kbps.m3u8
Due to this change we are facing issues with our current setup.
Is there any way to restore the previous file naming structure?

**Comment by yashtandon113** [comment, marked as answer]
The reason for changing the file name structure in v2.4.1 and later is to play the same resolution at different bit rates. The file name structure is configurable beginning with v2.4.3.
To achieve the previous version's file name structure, add or change the following property in the _**/usr/local/antmedia/webapps/appname/WEB-INF/red5-web.properties**_ file.

`settings.fileNameFormat=%r` ( by default it is `settings.fileNameFormat=%r%b` ) where

*%r = resolution*
*%b = bitrate*

After making the changes please do restart the server with `sudo service antmedia restart.`

**Reply by Mohit-3196** [reply]
Hi All,
Let me give an update on this, starting from Ant Media Sever v2.6.2, the application settings be it in Standalone Mode or Cluster mode can be changed directly from the Web Panel without having to edit the application configuration file red5-web.properties and restarting the server.

For more details check the application configuration document https://antmedia.io/docs/guides/configuration-and-testing/ams-application-configuration/

Thank you.
